templates/feature.html

```diff
@@ -82,44 +82,30 @@
           {% else %}
             Enabled by default
           {% endif %}
-        {% endif %} in <b>desktop Chrome {{ feature.meta.milestone_str }}</b>
+        {% endif %}
+        {% if feature.bug_url %}
+        (<a href="{{ feature.bug_url }}">launch bug</a>)
+        {% endif %}
+        in: </br>
+        {% if feature.shipped_milestone %}
+          - <b>Chrome for desktop release {{ feature.meta.milestone_str }}</b></br>
```

Comment:
I tried to use that. Then I'm going to need to add a few css lines as well as the current css with margin=0 does't fit the content. Is it okay to add some new classes to the css file as well?